src/sys/dune

```diff
@@ -1,4 +1,6 @@
-;; Tests of Semaphore.Counting
+;; Test of the Sys library
+
+;; this prevents the tests from running on a default build
```

Comment:
This comment should be removed since it describes an old rule which is no longer there